src/conditions/requiredBody.ts

```diff
@@ -0,0 +1,27 @@
+import { ConditionConfig } from '../config'
+import { PullRequestInfo } from '../models'
+import { ConditionResult } from '../condition'
+
+export default function hasRequiredLabels (
+  config: ConditionConfig,
+  pullRequestInfo: PullRequestInfo
+): ConditionResult {
+  if (config.requiredBodyRegex === undefined) {
+    return {
+      status: 'success'
+    }
+  }
+
+  const regexObj = new RegExp(config.requiredBodyRegex, 'ig')
+
+  if (regexObj.test(pullRequestInfo.body)) {
+    return {
+      status: 'success'
+    }
+  }
+
+  return {
+    status: 'fail',
+    message: `Required words were not found in body (${pullRequestInfo.body})`
```

Comment:
```suggestion
    message: `Required regular expression did not match body (\`${pullRequestInfo.body.replace('`', '\\`')}\`)`
```

Using `\`` in the regex is a likely use-case, so escaping this here would be nice. Also, it would be nice to test an example of a regex that includes `\`` and its resulting message .

Reply:
yeah agreed, not sure if we should print the body at all, wdyt? It can get quite long

Reply:
Hmm you're right. Removing it might be best. It is good to mention that this is about a regex not matching and not individual words.